src/app/auth/components/login/login.component.html

```diff
@@ -0,0 +1,10 @@
+<div class="login-wrapper">
+    <div class="login-form">
+      <h1 class="login-h1">Login</h1>
+      <label class="login-label" for="login">login</label>
+      <input type="text" class="login-input" #login (keyup)="onLoginChange(login.value)">
```

Comment:
It's bad practice to call onPasswordChange(password.value), onLoginChange(login.value) on each key up.
It's better to make login page as a form and assign values on submit button click.

Reply:
I changed it